backend/graph/nodes.py

```
@@ -0,0 +1,57 @@
+            return {"detected_lang": detected}
+
+    return {"detected_lang": "es"}
+
+
+def call_model(state: State):
+    system_prompt = get_rag_system_prompt(state.detected_lang)
+
+    if state.summary:
+        system_prompt += f"\n\nSummary of conversation earlier: {state.summary}"
+
+    messages = [SystemMessage(content=system_prompt)] + state.messages
```

Ruff (0.15.0)

[warning] Consider iterable unpacking instead of concatenation

Replace with iterable unpacking

(RUF005)